InterviewQuestionsTests/Chapters/ArraysStrings/ArraysStrings.Tests.cs

```diff
@@ -1,25 +1,33 @@
-﻿using InterviewQuestions.Chapters.ArraysStrings;
-
-namespace InterviewQuestionsTests.Chapters.ArraysStrings
-{
-    [TestClass]
-    public class ArraysStringsTests
-    {
-        [DataTestMethod]
-        [DataRow("abcdef", true)]
-        [DataRow("abcdea", false)]
-        [DataRow("abcdeA", true)]
-        [DataRow("abcdef g", true)]
-        [DataRow("abcdef b", false)]
-        public void isStringUniqTest(string input, bool expectedResult)
-        {
-            ArraysAndStringSolutions testObject = new ArraysAndStringSolutions();
-
-            bool output = testObject.isStringUniq(input);
-            bool outputSecond = testObject.isStringUniqSecondSolution(input);
-
-            Assert.AreEqual(expectedResult, output);
-            Assert.AreEqual(expectedResult, outputSecond);
-        }
-    }
-}
+﻿using InterviewQuestions.Chapters.ArraysStrings;
+
+namespace InterviewQuestionsTests.Chapters.ArraysStrings
+{
+    [TestClass]
+    public class ArraysStringsTests
+    {
+        [DataTestMethod]
+        [DataRow("abcdef", true)]
+        [DataRow("abcdea", false)]
+        [DataRow("abcdeA", true)]
+        [DataRow("abcdef g", true)]
+        [DataRow("abcdef b", false)]
+        public void isStringUniqTest(string input, bool expectedResult)
+
+        {
+            ArraysAndStringSolutions testObject = new ArraysAndStringSolutions();
+
+            bool output = testObject.isStringUniq(input);
+            bool outputSecond = testObject.isStringUniqSecondSolution(input);
+
+            
+            Assert.AreEqual(expectedResult, output);
+            Assert.AreEqual(expectedResult, outputSecond);    
+        }

```

Comment:
please add data test inputs to this test case as shown above. For now this test does not test anything. Also add assertions int he end of the tests to validate your function works as expected

Reply:
aabcccccaaa ->a2b1c5a3
aaabbcccddddee -> a3b2c3d4e2
abccdeff -> abccdeff
xxyyycdfaa -> xxyyycdfaa

add those test cases. left ones will be your input and right ones will be the expected results

Reply:
Please do not resolve the conversations by yourself.Add your thoughts as comment here.

Reply:
I still don't see that input of the functions. It should be seen something like this at beginning of the your test method.

[DataTestMethod]
[DataRow("aabcccccaaa ", a2b1c5a3)]
[DataRow("aaabbcccddddee ", a3b2c3d4e2)]

Reply:
I added the DataTestMethod.Do you mind if the methods have the same name?

Reply:
you can see the solution here 35616981a6ccca5bcc57ea6f305a72ca0a39f663

Reply:
Thanks